lib/public/views/Runs/formatRunDuration.js

```diff
@@ -27,16 +27,18 @@ export const formatRunDuration = (run, raw = false) => {
     if (runDuration === null || runDuration === undefined) {
         return '-';
     } else if (timeTrgEnd || timeO2End) {
-        return `${Math.floor(runDuration / 60000)} min${raw ? '' : getAsteriks(run)}`;
+        return `${formatDuration(runDuration)}${raw ? '' : _getAsteriks(run)}`;
     } else {
-        const time = timeTrgStart ? timeTrgStart : timeO2Start;
+        const timeStart = timeTrgStart == null || timeTrgStart === undefined ?
+            timeTrgStart :
+            timeO2Start;
```

Comment:
Don't you think that putting the symbol on the same line than the expression is more clear?

```js
const timeStart = timeTrgStart == null || timeTrgStart === undefined
    ? timeTrgStart
    : timeO2Start

Reply:
I think it does not matter that much but I changed it to be more consistent with the code.